http/headers/referrer-policy.json

```diff
@@ -5,11 +5,11 @@
         "support": false
       },
       "Chrome": {
-        "support": false,
+        "support": "56",
         "notes": ["See <a href=\"https://bugs.chromium.org/p/chromium/issues/detail?id=619228\">Chromium bug 619228</a>."]
```

Comment:
Done as you asked, https://github.com/mdn/browser-compat-data/pull/88/commits/5071e11251d4764292bc686b1af115eeeb920722